src/controllers/warehousecontroller.js

```diff
@@ -59,6 +59,7 @@ class WarehouseController extends Controller
     {
         // text heading
         $("#warehouse-used-capacity").html(GAME.model.warehouse.usedContainerCapacity());
+        $("#warehouse-max-capacity").html(GAME.model.warehouse.maxContainerCapacity());
```

Comment:
We had missed this when updating the view. Of course, the max capacity field could change also!